float64 left_pwm
float64 right_pwm
float64 duration
int32 action_num
int8 robot_id